Cargo.toml

```diff
@@ -7,3 +7,6 @@ repository = "https://github.com/dragostis/pest"
 documentation = "http://dragostis.github.io/pest/pest/"
 keywords = ["pest", "parser", "peg", "grammar"]
 license = "MPL-2.0"
+
+[dependencies]
+clap = "2.8.0"
```

Comment:
There is an example in handlebars that deals with both rustc_serialize and serde https://github.com/sunng87/handlebars-rust/blob/master/examples/render_file.rs. 

You can wrap most of main mod in a `#cfg(feature=...)` block so it won't compile if the feature is turned off.